src/invidious/views/preferences.ecr

```diff
@@ -57,6 +57,17 @@
                 </select>
             </div>
 
+            <% if !CONFIG.disabled?("dash") %>
+                <div class="pure-control-group">
+                    <label for="quality_dash"><%= translate(locale, "Preferred dash video quality: ") %></label>
+                    <select name="quality_dash" id="quality_dash">
+                        <% {"auto", "best", "4320p", "2160p", "1440p", "1080p", "720p", "480p", "360p", "240p", "144p", "worst"}.each do |option| %>
```

Comment:
All those options are available in the preferences. In your PR you say only auto best and worst should be available?

Reply:
I think he just implemented my comment and didnt update the description https://github.com/iv-org/invidious/pull/1572#issuecomment-748291375

Reply:
I think he just implemented what I was suggested in my comment (https://github.com/iv-org/invidious/pull/1572#issuecomment-748291375) and didn't update the description

Reply:
> I think he just implemented what I was suggested in my comment ([#1572 (comment)](https://github.com/iv-org/invidious/pull/1572#issuecomment-748291375)) and didn't update the description

Correct. An updated description would be:

The options are `auto` (the current and default behavior), `best`, `worst`, `4320p`, `2160p`, `1440p`, `1080p`, `720p`, `480p`, `360p`, `240p` and `144p`.
If the target height is not available the closest lower height is selected.

The UI is only updated once playback starts.